lib/picker.js

```diff
@@ -235,6 +235,16 @@ function PickerConstructor( ELEMENT, NAME, COMPONENT, OPTIONS ) {
                 $ELEMENT.addClass( CLASSES.active )
                 aria( ELEMENT, 'expanded', true )
 
+                // If the picker is within an iframe, set the picker height
+								function calculateTop() {
+										return (window.parent.document.documentElement.scrollTop || window.parent.document.body.scrollTop) + "px";
```

Comment:
The short answer is how the browser defines document.documentElement vs. document.body.  Some browsers may return undefined.